renovate.json

```diff
@@ -4,7 +4,8 @@
     "group:monorepos",
     "packages:stylelint",
     ":widenPeerDependencies",
-    ":label(renovate)"
+    ":label(renovate)",
+    ":semanticCommitScopeDisabled"
```

Comment:
> If you want to remove the semantic commit scope (e.g. use prefix chore: instead of chore(deps):), then add the preset ":semanticCommitScopeDisabled" to your extends array.
https://docs.renovatebot.com/semantic-commits/

よさそう